addons/account/static/src/components/section_and_note_fields_backend/section_and_note_backend.scss

```diff
@@ -2,40 +2,65 @@
 // The goal of this file is to contain CSS hacks related to allowing
 // section and note on sale order and invoice.
 
-table.o_section_and_note_list_view tr.o_data_row {
-    &.o_is_line_note {
-        font-style: italic;
-    }
-    &.o_is_line_section {
-        font-weight: bold;
-        background-color: #DDDDDD;
-        --table-bg: var(--300);
-    }
-    &.o_is_line_subsection {
-        --table-bg: var(--200);
-    }
-    &.o_is_line_section, &.o_is_line_subsection {
-        border-top: 1px solid #BBB;
-        border-bottom: 1px solid #BBB;
-        .o_list_section_options {
-            width: 1px;  // to prevent the column to expand
-            button {
-                padding: 0px;
-                background: none;
-                border-style: none;
-                display: table-cell;
-                cursor: pointer;
+table.o_section_and_note_list_view {
+    --o-section-border-color: #{$gray-500};
+    --o-section-hover-effect-transition: 0.1s ease-in-out;
+
+    tr {
+        &.o_is_line_note {
+            font-style: italic;
+        }
+
+        &.o_is_line_section, &.o_is_line_subsection {
+            --table-striped-bg: var(--table-bg-type);
+
+            .o_list_section_options {
+                width: 1px;  // to prevent the column to expand
+            }
+
+            td[name="price_subtotal"] {
+                color: $gray-700;
             }
         }
-    }
-    &.o_is_line_note,
-    &.o_is_line_section,
-    &.o_is_line_subsection {
-        td {
-            // There is an undeterministic CSS behaviour in Chrome related to
-            // the combination of the row's and its children's borders.
-            border: none !important;
+
+        &.o_is_line_section {
+            --table-hover-bg: #{rgba($body-emphasis-color, .08)};
+            --table-bg-type: #{rgba($body-emphasis-color, .06)};
+
+            .o_handle_cell .o_row_handle {
+                transition: opacity var(--o-section-hover-effect-transition);
+            }
+
+            &:where(:not(.o_selected_row)) {
+                div:has(> div.o_field_product_label_section_and_note_cell) {
+                    transition: transform var(--o-section-hover-effect-transition);
+                    position: absolute;
+                }
+
+                &:not(:hover, .o_dragged) {
```

Comment:
I can still drag the line around during edition.
Could we show the handle in this case as well? 

https://github.com/user-attachments/assets/f50b0d4d-84e4-4377-8144-224d79283881

Reply:
It's too be fixed on the new products as well. There seems to be a functional issue when drag & dropping the element in edition if the content is empty, it will delete the row. I think it should functionnally be restricted instead but it's probably not in the current scope of this PR. Wdyt?